### ISSUE TITLE
What change need make to compatible with Hi-fi GAN?

### ISSUE THREAD

**Issue by faranaziz** [mindslab-ai/cotatron#15]
I mean pre processing the mels. 
https://github.com/jik876/hifi-gan/issues/61

**Comment by seungwonpark** [comment]
If you're using the Hi-fi GAN (as shown in Assem-VC), then you should have a look at both mel calculation code and the configuration:
- mel calculation script: https://github.com/mindslab-ai/cotatron/blob/master/modules/mel.py
- config: https://github.com/mindslab-ai/cotatron/blob/master/config/global/default.yaml

You'll need to change the first one to https://github.com/jik876/hifi-gan/blob/master/meldataset.py#L49-L72, and the second one to https://github.com/jik876/hifi-gan/blob/master/config_v3.json#L17-L27. Hope that helps.

**Comment by faranaziz** [comment]
Thanks so much. 
Is this how you trained the new upcoming model?

**Comment by seungwonpark** [comment]
No. We changed the mel calculation script and configuration to match the Hifi-GAN’s version.

**Comment by faranaziz** [comment]
I make suggested change and get artifcat like reported here:
https://colab.research.google.com/gist/tulasiram58827/8a7660ab21ca3ea246141bbe4b0f87c0/hifigan_issue.ipynb

**Comment by Rashi2011** [comment]
I am trying to take mel from fastspeech2 and put that is hifiGAN to generate audio but all I am getting is noise . please suggest some Ideas.
